engine/client/gl_local.h

```diff
@@ -349,6 +349,72 @@ void R_LightForPoint( const vec3_t point, color24 *ambientLight, qboolean invLig
 int R_CountSurfaceDlights( msurface_t *surf );
 int R_CountDlights( void );
 
+
+/// <STROBE> // Move to strobe.h ?
+//#define SolidConsoleX 0
+//#define SolidConsoleY (y - scr_width->value * 3 / 4)
+//#define SolidConsoleW (scr_width->value)
+//#define SolidConsoleH (scr_width->value * 3 / 4)
+//extern wrect_t con_rect; //Float - Int incompatibility
+typedef struct _rectf_s
+{
+	float x, y, w, h;
+} _rectf_t;
+extern _rectf_t con_rect;
```

Comment:
Such generic definitions may be moved to "client.h" header, as con_rect can be used by other parts of engine.

Reply:
That would also require moving the _rectf definition to more generic header (client.h in this case). I did not do that because I intended to use _rectf only for strobe internal purposes.

I am not familiar with Xash3D so,

1) I am not sure if this definition would be ok to be in a more generic header for public / general uses.
2) There may also be a float / double rectangle struct definition in the engine. I have found similar wrect but it was int.

Reply:
It's just doesn't look right. I don't know can it be used elsewhere than strobing code. If it's intended to be private, you may move extern "_rectf_t" to file with your strobing code and don't show it for everyone(I mean, for code, yes :))